safekeeper/src/metrics.rs

```diff
@@ -163,6 +163,13 @@ pub static PARTIAL_BACKUP_UPLOADED_BYTES: Lazy<IntCounter> = Lazy::new(|| {
     )
     .expect("Failed to register safekeeper_partial_backup_uploaded_bytes_total counter")
 });
+pub static MANAGERS_RUNNING: Lazy<IntGauge> = Lazy::new(|| {
+    register_int_gauge!(
+        "safekeeper_managers_running",
```

Comment:
I'd prefer `_managers` in the suffix (as per https://prometheus.io/docs/practices/naming/). I think `safekeeper_active_managers` is a better name.

Also, not 100% sure about this, but we usually use `register_int_counter_pair!(` for similar counters, to count started/finished metrics separately, because it's often has useful data.